### DISCUSSION TITLE
Support of python3.9

### DISCUSSION THREAD

**Discussion by GuillaumeSeren** [qutebrowser/qutebrowser#5658] [Q&A] [answered]
Hello there,
I am working on the packaging of qutebrowser-1.13.1 for gentoo,
and I was wondering does qutebrowser support python3.9 ?

Thank you for qutebrowser it is so usefull to me !

**Comment by The-Compiler** [comment, marked as answer]
Thanks for taking care of the Gentoo package! :+1: Python 3.9 is [still in beta](https://www.python.org/dev/peps/pep-0596/), so while there are no new features permitted, some behavior changes could still occur.

That being said, qutebrowser was tested against Python 3.10 nightlies since about a month (7f1ced0e436ee73eac1303c1fd38eb751ecfcc05) and is now instead tested against the 3.9 beta as that made CI configuration a bit simpler (defd18d963fa1a79d3116a77664cc23c73dde602). Other than a minor issue with a development-only dependency (https://github.com/ionelmc/python-hunter/issues/87), I didn't run into any issues.

Thus, I [added](68f3778bfade54d421b8caf176f626f7c6a4d6d7) the Python 3.9 PyPI classifier, which now makes this official! :slightly_smiling_face:

**Reply by GuillaumeSeren** [reply]
Thank you @The-Compiler :)

**Reply by The-Compiler** [reply]
Quick update: [Python 3.9 RC 1 is out now](https://pythoninsider.blogspot.com/2020/08/python-390rc1-is-now-available.html), and things seem to work fine still. I guess at this point it's pretty clear that it should work fine with the final release as well.
